src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -1031,6 +1088,93 @@ define([
             ]);
         },
 
+        getPreviewDocument: function() {
+            if (!!this.previewWindow) {
+                return this.previewWindow.document;
+            } else {
+                return this.sandbox.dom.find('iframe', this.$preview).contents()[0];
+            }
+        },
+
+        handleChanges: function(changes) {
+            if (!!changes.reload) {
+                this.getPreviewDocument().location.reload();
+            } else {
+                // foreach property which was changed
+                for (var propertyName in changes) {
+                    if (changes.hasOwnProperty(propertyName)) {
+                        if (-1 !== propertyName.indexOf(',')) {
+                            this.handleSequence(propertyName, changes[propertyName]);
+                        } else {
+                            this.handleSingle(propertyName, changes[propertyName]);
+                        }
+                    }
+                }
+            }
+        },
+
+        handleSequence: function(propertyName, content) {
+            var sequence = propertyName.split(','),
+                filter = '',
+                elements,
+                nodeArray,
+                i, item, before = 0;
+            for (item in sequence) {
+                // check of integer
+                if (!/^\+?(0|[1-9]\d*)$/.test(sequence[item])) {
+                    before = sequence[item];
+                    filter += ' *[property="' + sequence[item] + '"]';
+                } else {
+                    filter += ' *[rel="' + before + '"]:nth-child(' + (parseInt(sequence[item]) + 1) + ')';
+                }
+            }
+            // find rdfa node
+            elements = this.getPreviewDocument().querySelectorAll(filter);
+            i = 0;
+            // foreach node
+            nodeArray = [].slice.call(elements);
+            nodeArray.forEach(function(element) {
+                // set content and highlight class
+                if (typeof content[i] !== 'undefined') {
+                    element.innerHTML = content[i];
+                } else {
+                    element.innerHTML = '';
+                }
+                // FIXME jump to element jump in from to: element.scrollIntoView();
```

Comment:
i confuses me to ... but i think i would like to say, that we could jump here to element ... the other i don`t understand ...